### PR TITLE
Implement create pipeline API

### PR DESCRIPTION
**Pull request by mondwan**
* Integrate with BaseManager
* Add doc about how to create a pipeline

Reference:
* https://docs.gitlab.com/ee/api/pipelines.html#create-a-new-pipeline

### CODE REVIEW

**Comment by mondwan** [comment]
Notes: I am not about to run `tox` with `py27` and `py34` by default.

File: docs/gl_objects/projects.py [hunk 1]
```diff
@@ -400,6 +400,12 @@
 pipeline = project.pipelines.get(pipeline_id)
 # end pipeline get
 
+# pipeline create
+pipeline = gl.project_pipelines.create({'project_id': 1, 'ref': 'master'})
+# or
+project.pipelines.create({'ref': 'master'})
```
Comment (gpocentek) [inline docs/gl_objects/projects.py]: Could you just do a minor edit for consistency: 

`````python
pipeline = project.pipelines.create({'ref': 'master'})
`````

**Comment by mondwan** [comment]
Sure. I have updated the document as requested

**Comment by gpocentek** [comment]
Thank you!

**Pull request merged**